### DISCUSSION TITLE
Momentum balance in the shocked shell for M1-67

### DISCUSSION THREAD

**Discussion by will-henney** [will-henney/globule-seminario#23] [Ideas]
In principle, the momentum in the radiation field and in the stellar wind can both contribute to the inner confinement of the shocked shell. 

For the radiation, the momentum absorbed will be proportional to $\tau L / c$, where $\tau$ is the optical depth of the shell. 

For the wind, the momentum absorbed will be proportional to $\eta L / c$, where $\eta$ is the momentum efficiency of the wind. 

For WR stars, we expect $\eta \simeq 1$, whereas we know that $\tau$ must be small based on the observed density and thickness.  Therefore, it is likely that the wind dominates
![CleanShot 2023-04-18 at 14 01 24@2x](https://user-images.githubusercontent.com/161264/232891995-0da06b25-c3a1-49d5-a9ef-ab2e1398a44d.jpg)

**Comment by RobeReyes** [comment]
For $\eta$ i found this paper of [Hamann et al. 1993](https://ui.adsabs.harvard.edu/abs/1993A%26A...274..397H/abstract) and maybe the values of the others paper came from this.
In table 5 they give a value of $\eta=6.7$, i don't understand this momentum and $\eta$

**Comment by RobeReyes** [comment]
i know that $\eta$ is the momentum efficiency, but i got confused with this and units of $L_*$

**Comment by will-henney** [comment]
eta is often larger than unity for WR winds, since they are optically
thick. So 6.7 seems a reasonable value

We can discuss this in the class today


  Dr William Henney, Instituto de Radioastronomía y Astrofísica,
  Universidad Nacional Autónoma de México, Campus Morelia
